components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -1528,8 +1531,35 @@ void RewardsServiceImpl::GetAddresses(const GetAddressesCallback& callback) {
     return;
   }
 
-  bat_ledger_->GetAddresses(base::BindOnce(&RewardsServiceImpl::OnGetAddresses,
-        AsWeakPtr(), callback));
+  bat_ledger_->GetAddressesLimitedCountries(
+      base::BindOnce(&RewardsServiceImpl::OnGetAddressesLimitedCountries,
+              AsWeakPtr(), callback));
+}
+
+void RewardsServiceImpl::OnGetAddressesLimitedCountries(
+    const GetAddressesCallback& callback,
+    const std::vector<std::string>& country_chars) {
```

Comment:
`country_chars` is confusing as the vector contains strings, how about `country_codes`